### DISCUSSION TITLE
Serene (public) template for v8?

### DISCUSSION THREAD

**Discussion by LSNicholls** [serenity-is/Serenity#7101] [General]
Hi Serenity folks,

I would like to re-do my WinAuth Sample (see [this](https://github.com/orgs/serenity-is/discussions/7100)), which is public, without the mix of versions of things that it currently contains, IOW start it from a "fresh" Serene Template set up for Asp.Net Core v8, the latest and greatest Serenity core libraries, the later version of sergen, etc.  If I try to upgrade everything in-place, things really get messy.

Not trying to rush you, but do you have a schedule for when the VSIX will be available?  

Thanks as always for what you do.

**Comment by VictorTomaili** [comment]
Currently we didn't have any schedule

**Reply by LSNicholls** [reply]
Thank you for the reply, @VictorTomaili.

**Comment by volkanceylan** [comment]
We'll release it eventually but anyone who downloads it from marketplace etc. will not be able to make it run if they don't have latest VS updates installed as VS2022 supports .NET 6 by default.
